File: src/main/java/net/paulem/btm/utils/PluginUtils.java
package net.paulem.btm.utils;

import net.paulem.btm.BetterMending;
import net.paulem.btm.listeners.extendables.ManagersListener;

public class PluginUtils {
    public static void reloadConfig() {
        BetterMending.getInstance().reloadConfig();

        if(BetterMending.getPlayerConfig() != null) BetterMending.getPlayerConfig().reload();

        ManagersListener.reloadConfig(BetterMending.getConf());
    }
}

File: src/main/kotlin/net/paulem/btm/utils/PluginUtils.kt
package net.paulem.btm.utils

import net.paulem.btm.BetterMending
import net.paulem.btm.listeners.extendables.ManagersListener

object PluginUtils {
    @JvmStatic
    fun reloadConfig() {
        BetterMending.instance.reloadConfig()
        BetterMending.instance.reloadModifiables()

        BetterMending.playerConfig.reload()

        ManagersListener.reloadConfig(BetterMending.instance.getConfig())
    }

    @JvmStatic
    fun parseConfigText(path: String, def: String): String {
        return BetterMending.instance.getConfig().getString(
            path,
            def
        )!!.replace("&", "§")
    }
}